Week1/assignments/usmaila-abdoul/Palindrome.js

```diff
@@ -0,0 +1,15 @@
+function Palindrome(string) {
+    // 1 change all strings to lower case
+    // 2 reverse the link and compare with origin string to see if its a palindrome
+    // 3 return true if its a palindrome
+    let lowercaseString = string.toLowerCase();
+    let newString = lowercaseString.split('').reverse().join('');
+
+    if (newString === lowercaseString) {
```

Comment:
Line 8 to 12 could be replaced with `return newString === lowercaseString`
Also remove the solution to `Staircase` from this PR